HexFiend_2.xcodeproj/project.pbxproj

```diff
@@ -2217,7 +2218,7 @@
 					"@executable_path/../Frameworks",
 					"@loader_path/../Frameworks",
 				);
-				MACOSX_DEPLOYMENT_TARGET = 10.9;
+				MACOSX_DEPLOYMENT_TARGET = "$(RECOMMENDED_MACOSX_DEPLOYMENT_TARGET)";
```

Comment:
Maybe I'm missing something, but where is RECOMMENDED_MACOSX_DEPLOYMENT_TARGET set, and what is it set to?

Also, how was it added to a bunch of build configs that had no deployment target before? Shouldn't this be just set at the project level and the targets all be clear?

Reply:
Xcode defines/sets the "recommended" version. Xcode 14.1 sets it to 10.14.6 if I last recall. Future Xcode versions can change this, so it may technically be a moving target.

Reply:
As for the second part of my original question, I discovered that this was done automatically by Xcode as part of 'Update to recommended settings'.

Reply:
> What I don't understand is why it is set to 10.14.6 if the actual minimum supported by Xcode 14 is 10.13. Should we manually set to 10.13 instead so support for 10.13 isn't dropped unnecessarily? 

this just means Apples “recommended“ is different than the “minimum” supported for whatever reason. Up to the project to choose what to set it as. Also up to the project if it wants to set it to a variable that is subject to change in the future.

If the project wants to use Swift without bloating the app for example, dropping 10.13 could make sense.

Reply:
> this just means Apples “recommended“ is different than the “minimum” supported for whatever reason.

Good point.

Ah, yes. I forgot what version of macOS was the first to get Swift 5 support and ABI stability. Hard to believe Swift 5 first came out so long ago now! With Swift in mind 10.14.6 makes a lot more sense as the recommended minimum.